server/services/mongodb-remote/src/test/java/com/mongodb/stitch/server/services/mongodb/remote/internal/SyncMongoClientIntTests.kt

```diff
@@ -899,6 +902,27 @@ class SyncMongoClientIntTests : BaseStitchServerIntTest() {
         }
     }
 
+    @Test
+    fun testConfigure() {
+        val testSync = getTestSync()
+
+        (mongoClient as RemoteMongoClientImpl).dataSynchronizer.enableSyncThread()
```

Comment:
I'm not quite positive, but it seems like the expose and implementation of `.enableSyncThread()` is not strictly necessary. You can use manual sync passes if you need to.

Reply:
Alright. This makes more sense. It will slightly change the tests, but yes, if we address your comment below, we can remove the `enableSyncThread` method.